http/headers/Permissions-Policy.json

```diff
@@ -1309,7 +1309,7 @@
               "chrome_android": "mirror",
               "edge": "mirror",
               "firefox": {
-                "version_added": "preview"
+                "version_added": "126"
```

Comment:
This matches the corresponding Bugzilla issue. Related docs work tracking in https://github.com/mdn/content/issues/33047